include/libsemigroups/dot.hpp

```diff
@@ -61,17 +63,100 @@ namespace libsemigroups {
       return std::to_string(std::forward<Thing>(thing));
     }
   }  // namespace detail
-
+  //
+  //! \defgroup dot_group Visualisation
+  //!
+  //! This page contains links to the various classes and functions in
+  //! ``libsemigroups`` for visualising instances of some of the other objects
+  //! implemented in ``libsemigroups``.
+  //!
+  //! The class Dot facilitates the creation and rendering of graph
+  //! descriptions in the [DOT][] language of the [Graphviz][] graph drawing
+  //! software.
+  //!
+  //! The function \ref dot can be used to produce initial Dot objects for
+  //! various of the data structures and algorithms in ``libsemigroups``:
+  //!
+  //! * \ref dot(AhoCorasick&)
+  //! * \ref dot(WordGraph<Node> const& wg)
```

Comment:
This doesn't seem to create a link for me, though I can't see why not. Would you expect it to?

Reply:
That's intentional at present, there's no such do to link to.